src/app/shared/palette/order-summary/read-order-summary-field.html

```diff
@@ -0,0 +1,14 @@
+<div class="order-summary-title">Order Summary</div>
+<table>
+    <thead>
+      <tr><td>Code</td><td>Description</td><td>Amount</td></tr>
+    </thead>
+    <tbody>
+        <tr ccdReadOrderSummaryRow *ngFor="let feeValue of getFees()" [feeValue]="feeValue"></tr>
```

Comment:
I think would be better move the <tr> inside the template of ccdReadOrderSummaryRow. It's ccdReadOrderSummaryRow that should be in control of how it renders itself
And here would be:
<ccdReadOrderSummaryRow *ngFor....>

Reply:
This will not work hence me spending some time researching to attach on attribute instead of tag. The table styling would never work in any browser if we have custom tags wrapping tr->td tags. It has to be table->tbody->tr->td. table->tbody->custom-tag->tr->td will never work.